src/main/java/it/pagopa/transactions/client/PaymentGatewayClient.java

```diff
@@ -699,19 +700,26 @@ private String encodeMdcFields(AuthorizationRequestData authorizationData) {
 
     private Either<RedirectConfigurationException, URI> getRedirectUrlForPsp(
                                                                              String pspId,
-                                                                             String paymentTypeCode
+                                                                             String paymentTypeCode,
+                                                                             String channelCode
```

Comment:
re-order parameters to be touchpoint, pspId and then paymentTypeCode so that they reflects the searching order

Reply:
Done.